plugins/w3c_logs.yaml

```diff
@@ -74,6 +110,7 @@ template: |
       storage: file_storage
       start_at: '{{ .start_at }}'
       max_concurrent_files: {{ .max_concurrent_files }}
+      delete_after_read: {{ .delete_after_read }}
```

Comment:
Delete after read only works if `start_at` is at beginning. We should add that to the description and also add a conditional here.